modules/kubefed-check-cluster-status.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+// * kubefed/kubefed-resource-federation.adoc
+
+[id="kubefed-check-cluster-status_{context}"]
+= Checking the status of federated clusters
+
+.Procedure
+
+. Check the status of the joined clusters by using the following command:
++
+----
+$ kubectl get kubefedclusters -n <namespace>
+   NAME                   READY     AGE
+   <cluster-name>    True           7s
+----
++
+[NOTE]
+====
+The default namespace is `kube-federation-system`.
```

Comment:
Need to check with Sohan or Aniket, for OCP the default namespace is "kube-federation-system" or "openshift-federation-system".

Reply:
Per the bug: https://bugzilla.redhat.com/show_bug.cgi?id=1746943, this is no need in OCP 4.2.0